src/Controls/src/Core/Platform/iOS/Extensions/FormattedStringExtensions.cs

```diff
@@ -5,10 +5,12 @@
 using Microsoft.Maui.Controls.Internals;
 using Microsoft.Maui.Graphics;
 using System;
+using System.Linq;
```

Comment:
How hard would it be to remove linq from this code?

Reply:
Addressed!